tac/aea/mail/oef.py

```diff
@@ -30,11 +30,14 @@
 from oef.core import OEFProxy
 from oef.messages import OEFErrorOperation, CFP_TYPES, PROPOSE_TYPES
 from oef.proxy import OEFNetworkProxy
+from oef.schema import Description
 
-
-from tac.aea.mail.protocol import Envelope
 from tac.aea.mail.base import Connection, MailBox
-from tac.aea.mail.messages import OEFMessage, FIPAMessage, ByteMessage
+from tac.aea.mail.messages import OEFMessage, FIPAMessage, SimpleMessage
+from tac.aea.mail.protocol import Envelope, DefaultJSONSerializer
+from tac.aea.protocols.default.serialization import SimpleSerializer
```

Comment:
Also here, a mismatch in naming `simple` versus `default`. I'd be tempted to use `Default` throughout.